exporter/dynatraceexporter/metrics_exporter.go

```diff
@@ -109,19 +121,16 @@ func (e *exporter) serializeMetrics(md pdata.Metrics) []string {
 			for k := 0; k < metrics.Len(); k++ {
 				metric := metrics.At(k)
 
-				var l []string
-				switch metric.DataType() {
-				case pdata.MetricDataTypeNone:
-					continue
-				case pdata.MetricDataTypeGauge:
-					l = serialization.SerializeNumberDataPoints(e.cfg.Prefix, metric.Name(), metric.Gauge().DataPoints(), e.defaultDimensions)
-				case pdata.MetricDataTypeSum:
-					l = serialization.SerializeNumberDataPoints(e.cfg.Prefix, metric.Name(), metric.Sum().DataPoints(), e.defaultDimensions)
-				case pdata.MetricDataTypeHistogram:
-					l = serialization.SerializeHistogramMetrics(e.cfg.Prefix, metric.Name(), metric.Histogram().DataPoints(), e.defaultDimensions)
+				metricLines, err := serialization.SerializeMetric(e.cfg.Prefix, metric, e.defaultDimensions, e.staticDimensions, e.prevPts)
+
+				if err != nil {
+					e.logger.Sugar().Errorf("failed to serialize %s %s, %s", metric.DataType().String(), metric.Name(), err.Error())
```

Comment:
```suggestion
					e.logger.Sugar().Errorf("Failed to serialize %s '%s': %s", metric.DataType().String(), metric.Name(), err.Error())
```

Reply:
Seems like idiomatic go style is to keep lowercase for error strings
![image](https://user-images.githubusercontent.com/1612643/132031129-f458af1b-157c-4eb0-9a53-bf99c24be47e.png)

Reply:
I see, I just noticed it's inconsistent with the log in line 132 then.